test/test_misc.py

```diff
@@ -522,6 +529,29 @@ async def test_already_paused_client_during_wait_for_servers_shutdown(bouncer):
                 await task
 
 
+@pytest.mark.skipif(
+    "not USE_UNIX_SOCKETS", reason="This test does not apply to non unix sockets"
+)
+def test_shutdown_wait_for_clients(bouncer):
+    """
+    Test that after issu
+    """
+    with bouncer.cur() as cur, bouncer.admin_runner.cur():
+        cur.execute("SELECT 1")
+        bouncer.admin("SHUTDOWN WAIT_FOR_CLIENTS")
+
+        time.sleep(2)
+        bouncer.sql(query=";", host=bouncer.config_dir)
+        with pytest.raises(psycopg.errors.OperationalError):
+            bouncer.sql(query=";", host="127.0.0.1")
```

Comment:
Let's use the `bouncer.test(...)` method instead. Also for the other places. Maybe in another PR we could replace the `select 1` that that does internally with this empty query.
```suggestion
            bouncer.test(host="127.0.0.1")
```

Reply:
Went ahead and opened #1306 to change the test() functions to use null queries. Will change the code in this PR to use that instead of the sql() function.

Reply:
These are all changed to use the main test function.